app.js

```diff
@@ -0,0 +1,23 @@
+var express = require('express');
+var trucks = require('./trucks');
```

Comment:
This module isn't necessary, and because it's missing, it's causing an error when trying to run your server.